lib/utils.js

```diff
@@ -13,16 +15,19 @@ const internals = {};
 
 module.exports.log = function (err, options, callback) {
 
-    const log = Fs.createWriteStream(options.logPath, options.writeStreamOptions);
-    const formattedErr = {
-        message: err.message,
-        stack: err.stack,
-        timestamp: Date.now()
-    };
+    Mkdirp(Path.dirname(options.logPath), () => {
```

Comment:
Well a few things...

1. The calling function doesn't make use of it.

2. It seems a bit inconsequential seeing as how if we've gotten this far then we've an uncaught exception and the process is about to exit.

3. Adding the error handling code to the log function is obviously easy, but what do you think the plugin's calling function should do with the error? 

4. Also I'm not sure how to create a good cross-platform test for the err handling code. Any thoughts on that?

I'll add the error handling code in a few minutes but it'll take me longer to devise a test.